backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -273,4 +279,63 @@ public static Extension convertToTribalAffiliationExtension(String tribalAffilia
     }
     return null;
   }
+
+  public static Observation convertToObservation(
+      Result result, boolean corrected, String correctionReason) {
+    if (result != null && result.getDisease() != null) {
+      return convertToObservation(
+          result.getDisease().getLoinc(),
+          result.getDisease().getName(),
+          result.getResultLOINC(),
+          corrected,
+          correctionReason,
+          Objects.toString(result.getInternalId(), ""));
+    }
+    return null;
+  }
+
+  public static Observation convertToObservation(
+      String diseaseCode,
+      String diseaseName,
+      String resultCode,
+      boolean corrected,
+      String correctionReason,
+      String id) {
+    var observation = new Observation();
+    observation.setId(id);
+    observation.setStatus(ObservationStatus.FINAL);
+    addCode(diseaseCode, diseaseName, observation);
+    addValue(resultCode, observation);
+    addCorrectionNote(corrected, correctionReason, observation);
+    return observation;
+  }
+
+  private static void addCorrectionNote(
+      boolean corrected, String correctionReason, Observation observation) {
+    if (corrected) {
+      observation.setStatus(ObservationStatus.CORRECTED);
```

Comment:
I think we may need to add a bit more nuance to this status. Internally, we have three different statuses for TestEvents - original, corrected, and removed. I think that corrected corresponds to this ObservationStatus of CORRECTED, but removed might better correspond to an ObservationStatus of ENTEREDINERROR (or maybe CANCELLED). 

See [FHIR docs](https://build.fhir.org/valueset-observation-status.html#expansion) and [Javadoc](https://hapifhir.io/hapi-fhir/apidocs/hapi-fhir-structures-r4/org/hl7/fhir/r4/model/Observation.ObservationStatus.html)